public/pages/ChangePolicy/components/ChangeManagedIndices/ChangeManagedIndices.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { Component } from "react";
+import { EuiSpacer, EuiText, EuiComboBox, EuiFormRow } from "@elastic/eui";
+import { toastNotifications } from "ui/notify";
+import { ContentPanel } from "../../../../components/ContentPanel";
+import { ManagedIndexService } from "../../../../services";
+import { ManagedIndexItem, State } from "../../../../../models/interfaces";
+
+interface ChangeManagedIndicesProps {
+  managedIndexService: ManagedIndexService;
+  selectedManagedIndices: { label: string; value?: ManagedIndexItem }[];
+  selectedStateFilters: { label: string }[];
+  onChangeManagedIndices: (selectedManagedIndices: { label: string; value?: ManagedIndexItem }[]) => void;
+  onChangeStateFilters: (stateFilter: { label: string }[]) => void;
+  managedIndicesError: string;
+}
+
+interface ChangeManagedIndicesState {
+  managedIndicesIsLoading: boolean;
+  managedIndices: { label: string; value?: ManagedIndexItem }[];
+  stateFilterSearchValue: string;
+}
+
+export default class ChangeManagedIndices extends Component<ChangeManagedIndicesProps, ChangeManagedIndicesState> {
+  state = {
+    managedIndicesIsLoading: false,
+    managedIndices: [],
+    stateFilterSearchValue: "",
+  };
+
+  async componentDidMount(): Promise<void> {
+    await this.onManagedIndexSearchChange("");
+  }
+
+  onManagedIndexSearchChange = async (searchValue: string): Promise<void> => {
+    const { managedIndexService } = this.props;
+    this.setState({ managedIndicesIsLoading: true, managedIndices: [] });
+    try {
+      // only bring back the first 10 results descending by name
+      const queryParamsString = `?from=0&size=10&search=${searchValue}&sortDirection=desc&sortField=name`;
+      const managedIndicesResponse = await managedIndexService.getManagedIndices(queryParamsString);
+      if (managedIndicesResponse.ok) {
```

Comment:
I see in below you're accessing `managedIndicesResponse.response.managedIndices` shouldn't this be `managedIndicesResponse.response.ok` ?